infrastructure/templates/datafactory/template.json

```diff
@@ -47,10 +47,10 @@
     "actionGroupAlerts": {
       "type": "string"
     },
-    "slackWebhook": {
+    "slackAppToken": {
```

Comment:
I saw Ben was suggesting also fetching this variable from keyvault directly in this template rather than using template params, by adding something similar to this to the `variables` section:

`"ees-admin-govuknotify-api-key": "[resourceId('Microsoft.KeyVault/vaults/secrets', variables('keyVaultName'), 'ees-admin-govuknotify-api-key')]",`